### PR TITLE
Add additionalPaths parameter

### PR DESCRIPTION
**Pull request by KonstantinBozhkov**
I have 150k+ pages on my project, and I have to use fallback: true to avoid rendering them all. I can not use `additionalSitemaps` because I do not know the exact number of pages to divide them into different sitemap.

I suggest you consider my solution

### CODE REVIEW

**Comment by sheikalthaf** [comment]
@iamvishnusankar Any update on this PR ?. I also need to add additional paths which is not available in static paths

**Comment by KonstantinBozhkov** [comment]
@iamvishnusankar can you provide a reason for the closure? I spent my time improving this project instead of writing my implementation. And after a month of waiting, you just close the PR without even commenting. It is obvious that this functionality is needed not only for me (@sheikalthaf).

**Comment by iamvishnusankar** [comment]
@KonstantinBozhkov Thanks a lot for your contribution. I'm extremely sorry it was just an accident. I was really busy with my other projects. I just got some spare time today and was reviewing all PR. I accidentally closed your PR when bulk closing a lot of depadabot minor version bumps. Just re-opened and I'll review it now.

File: README.md [hunk 1]
```diff
@@ -89,6 +89,8 @@ Above is the minimal configuration to split a large sitemap. When the number of
 | outDir (optional)                              | All the generated files will be exported to this directory. Default `public`                                                                                                                                                                                                                                                                                      | string         |
 | transform (optional)                           | A transformation function, which runs **for each** `relative-path` in the sitemap. Returning `null` value from the transformation function will result in the exclusion of that specific `path` from the generated sitemap list.                                                                                                                                  | async function |
 
+| additionalPaths (optional) | A function that returns a list of additional paths to be added to the general list. | async function |
```
Comment (iamvishnusankar) [inline README.md:92]: Screenshot 2021-08-05 at 12 34 29 PM

Formatting issue.

Comment (KonstantinBozhkov) [reply README.md]: Fixed

**Review by iamvishnusankar — COMMENTED** [review]
@KonstantinBozhkov Thanks a lot for your contribution. Merged! :)

**Comment by KonstantinBozhkov** [comment]
Thank you

**Pull request merged**